### DISCUSSION TITLE
IPEA calculation: GFN1-xTB vs GFN2-xTB

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#679] [Q&A] [answered]
Hello,

I am testing the IPEA calculation for some molecules.

I optimized the structure first, and then run the other properties calculations.

In case of IPEA, adding the options `--vomega --vipea`, the output file shows that GFN1-xTB was used instead GFN2-xTB.

Forcing to use GFN2-xTB, I notice that the results are slightly different:
```
                                         GFN1-xTB     GFN2-xTB                
delta SCC IP (eV):                         5.3709       5.0636
delta SCC EA (eV):                         3.1036       2.6170
Global electrophilicity index (eV):        3.9593       3.0140
```

My questions:
1-) Why is GFN1-xTB as default for IPEA calculations whereas GFN2-xTB is default for all other calculations? 
2-) Which result should I use? (all the other calculations are done with GFN2-xTB)

Best regards,

Camps

**Comment by haneug** [comment, marked as answer]
Hi Camps,

1) It is not GFN1-xTB but IPEA-xTB that is used. IPEA-xTB employs the GFN1-xTB Hamiltonian, but with a special purpose parameterization for IPs and EAs and it comes with additional diffuse basis functions for a better description of EAs. 

2) Both methods GFN2-xTB and IPEA-xTB perform similar. In the gas phase, IPEA-xTB should be slightly better. But when you employ a solvation model you should stick with GFN2-xTB. It is important that you employ the shift for the self-energy (–4.8455 eV) for all IPs and EAs calculated with GFN2-xTB or IPEA-xTB.

Best,
Hagen

**Reply by icamps** [reply]
Thank you very much.
